v21.1/schema-design-database.md

```diff
@@ -10,27 +10,13 @@ This page provides best-practice guidance on creating databases, with a couple e
 For reference documentation on the `CREATE DATABASE` statement, including additional examples, see the [`CREATE DATABASE` syntax page](create-database.html).
 {{site.data.alerts.end}}
 
-<div class="filters filters-big clearfix">
-  <button class="filter-button" data-scope="local">Local</button>
-  <button class="filter-button" data-scope="cockroachcloud">CockroachCloud</button>
-</div>
-
 ## Before you begin
 
 Before reading this page, do the following:
 
-<ul>
-  <li>
-    <a href="install-cockroachdb.html">Install CockroachDB.</a>
-  </li>
-  <li>
-    <a class="filter-content" data-scope="local" href="secure-a-cluster.html">Start a local CockroachDB cluster.</a>
-    <a class="filter-content" data-scope="cockroachcloud" href="../cockroachcloud/create-your-cluster.html">Create a CockroachCloud cluster.</a>
-  </li>
-  <li>
-    <a href="schema-design-overview.html">Review the database schema objects.</a>
-  </li>
-</ul>
+- [Install CockroachDB](install-cockroachdb.html).
+- [Start a local cluster](secure-a-cluster.html), or [create a CockroachCloud cluster](../cockroachcloud/create-your-cluster.html).
```

Comment:
Can you make this same change to all of the other before you begin sections in this guide?